edx-platform/pearson-pathways-theme/lms/templates/footer.html

```diff
@@ -28,63 +28,78 @@
               <img alt="${_('organization logo')}" src="${footer['logo_image']}">
             </a>
           </p>
-          <h4>Empowering people to make progress in their lives through learning</h4>
+          <h4>Empowering people to make <br>
+            progress in their lives through <br>
+            learning</h4>
```

Comment:
Are the 'br' tags required? @anfbermudezme

Reply:
Yes because if we don't have de 'br' tags, the text looks like:

![Selection_222](https://user-images.githubusercontent.com/30726391/146418689-4180d37b-c409-4414-8287-b77430505fab.png)

And in figma the text is required like:

![Selection_223](https://user-images.githubusercontent.com/30726391/146418763-58b0b5b3-9620-4a9c-a58d-804ca6623435.png)